message_definitions/v1.0/uAvionix.xml

```diff
@@ -82,6 +82,18 @@
       <entry value="6" name="UAVIONIX_ADSB_OUT_DOWNED_AIRCRAFT_EMERGENCY"/>
       <entry value="7" name="UAVIONIX_ADSB_OUT_RESERVED"/>
     </enum>
+    <enum name="MAV_CMD">
+      <entry value="54132" name="MAV_CMD_DO_START_ADSB_OUT_IDENT" hasLocation="false" isDestination="false">
```

Comment:
That's a strange number to use.

Perhaps this `MAV_CMD` range could start at "10000" in the same way the uavionix messages do?  @hamishwillee thoughts?

Reply:
yup, it was quite a keyboard smash number. I'll change it to a 10k number.

Reply:
Sorry, missed that. No problem by me as long as new number isn't a clash.